examples/tree_viz.rs

```diff
@@ -1,36 +1,103 @@
-use argh::FromArgs;
-use blart::{
-    visitor::{DotPrinter, DotPrinterSettings},
-    AsBytes, NoPrefixesBytes, TreeMap,
-};
-use std::{
-    error::Error,
-    ffi::CString,
-    fmt::Display,
-    fs::OpenOptions,
-    io::{self, BufRead, BufReader, BufWriter, Write},
-    path::{Path, PathBuf},
-};
-
-#[derive(FromArgs)]
-/// TREES
-struct TreeToDotArgs {
-    /// optional delimiter to split key from value on each file line
-    #[argh(option)]
-    delimiter: Option<char>,
-
-    /// input to read keys from an external file
-    #[argh(positional)]
-    input_file: PathBuf,
-
-    /// where to output the tree diagram
-    ///
-    /// To output to stdout, use '_'.
-    #[argh(positional)]
-    output_location: String,
+#[cfg(feature = "std")]
```

Comment:
Instead of making this entire file `cfg(feature = "std")` could you add a `required-features = ["std"]` ([docs](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-required-features-field)) and make this `tree_viz` example an explicit entry under `[[examples]]` in the `Cargo.toml`?